src/luger_utils.erl

```diff
@@ -34,8 +34,11 @@ single_line(Msg, false) ->
 single_line(Msg, true) ->
     lists:join(" ", binary:split(iolist_to_binary(Msg), [<<"\n">>, <<" ">>], [global, trim_all])).
 
-message(Msg, SingleLine) ->
-    trunc(2048, single_line(Msg, SingleLine)).
+message(Msg, SingleLine, MaxLen) ->
+    case MaxLen of
+      undefined -> single_line(Msg, SingleLine);
+      _ -> trunc(MaxLen, single_line(Msg, SingleLine))
+    end.
```

Comment:
It's generally cleaner to pattern match on the function head instead of with `case` statements:
```suggestion
message(Msg, SingleLine, undefined) ->
    single_line(Msg, SingleLine);
message(Msg, SingleLine, MaxLen) ->
    trunc(MaxLen, single_line(Msg, SingleLine)).
```